data-es7.js

```diff
@@ -1190,6 +1190,63 @@ exports.tests = [
   res: {
     babel:       true,
   }
+},
+{
+  name: 'Observable',
+  category: 'proposal',
+  significance: 'large',
+  link: 'https://github.com/zenparsing/es-observable',
+  'subtests': {
+    'basic support': {
+      exec: function () {/*
+        return typeof Observable !== 'undefined';
+      */}
+    },
+    'Symbol.observable well known symbol': {
+      exec: function () {/*
+        return typeof Symbol.observable === 'symbol';
+      */}
+    },
+    'Observable.prototype.subscribe': {
+      exec: function () {/*
+        return 'subscribe' in Observable.prototype;
+      */}
+    },
+    'Observable.prototype.forEach': {
+      exec: function () {/*        
+        var o = new Observable();
+        return 'forEach' in Observable.prototype && o.forEach(function(e){return true}) instanceof Promise;
+      */}
+    },
+    'Observable.prototype.filter': {
+      exec: function () {/*
+        var o = new Observable();
+        return 'filter' in Observable.prototype && o.filter(function(e){return true}) instanceof Observable;
+      */}
+    },
+    'Observable.prototype.map': {
+      exec: function () {/*
+        var o = new Observable();
+        return 'map' in Observable.prototype && o.map(function(e){return e}) instanceof Observable;
+      */}
+    },
+    'Observable.prototype.@@observable': {
```

Comment:
Spec-shorthand like `@@` isn't used in the compat-table if there's a userland equivalent form - in this case, that's `Observable.prototype[Symbol.observable]`.